lib/system-windows-panel.coffee

```diff
@@ -0,0 +1,102 @@
+{ScrollView} = require 'atom-space-pen-views'
+SettingsPanel = require './settings-panel'
+Registry = require 'winreg'
+Path = require 'path'
+
+exeName = Path.basename(process.execPath)
+appPath = "\"#{process.execPath}\""
+appName = exeName.replace('atom', 'Atom').replace('beta', 'Beta').replace('.exe', '')
+
+fileHandlerRegistry = {
+  key: "\\Software\\Classes\\Applications\\#{exeName}",
+  parts: [{key: 'shell\\open\\command', name: '', value: "#{appPath} \"%1\""}]
+}
+
+contextRegistryParts = [
+    {key: 'command', name: '', value: "#{appPath} \"%1\""},
+    {name: '', value: "Open with #{appName}"},
+    {name: 'Icon', value: "#{appPath}"}
+]
+
+contextFileRegistry = {
```

Comment:
Since there is no compile time we do... In a lot of places we use CamelCase but that's actually not standard. In JS these kinds of constants declared at the top of the file are typically ALL_CAPS